lib/ansible/modules/system/user.py

```diff
@@ -514,7 +526,13 @@ def modify_user_usermod(self):
 
         if self.expires:
             cmd.append('--expiredate')
-            cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+            try:
```

Comment:
hi @nitzmahone , thanks for reviewing.
I tested again and it works without this try catch. I'm no python developer, so I just wanted to be sure no exception is unhandled if someone uses "expire" with a string or an empty value (I did not check thoroughly where execptions get caught outside of this position, sry).
I changed it in my fork.
thanks again!

/edit: at first tried to use only self.expire (no extra variable)